.github/ISSUE_TEMPLATE/propose-talk-workshop.md

```diff
@@ -39,12 +39,12 @@ One can submit a talk on anything related to Python or Open Source and below are
 <!-- Please specify the level of audience - Begineer/Internediate/Advance -->
 
 **Speaker Bio**
-<!-- Please do include the following things
-1. Speaker Bio (Brief)
-2. Company/College
-3. Email
-4. Years of Exp
- -->
+<!-- Please do include the following things-
+1. Speaker Bio (Brief)-
+2. Company/College-
+3. Email-
+4. Years of Exp -->
```

Comment:
let's move `-->` this to the next line and have a hyphen here `-`

Reply:
Moving --> to the next line introduces too much complexity to parse out the details we want.